src/WebJobs.Script/Environment/IEnvironment.cs

```diff
@@ -20,7 +20,7 @@ public interface IEnvironment
         string GetEnvironmentVariable(string name);
 
         /// <summary>
-        /// Creates, modifies, or deletes an environment variable stored in the current <see cref="IEnvironment"/>
+        /// Creates, modifies, or deletes an environment variable stored in the current <see cref="IEnvironment"/>.
```

Comment:
undo - to avoid format changes in the PR

Reply:
Changed back👍